docs and test: ignore_error_inputs (#2)
megaease/telegraf: docs/CONFIGURATION.md

  translates by calling external programs snmptranslate and snmptable,
  or "gosmi" which translates using the built-in gosmi library.

- **ignore_error_inputs**:

>   If set to true, Discard the Inputs that are error during initialization.

Reviewer: ```suggestion
  If set to true, discard the input plugins that produce the error during initialization.
```

>   If set to false, When there is an Input exception during initialization, the Telegraf program will exit directly.

Reviewer: ```suggestion
  If set to false, the program will exit when an input plugin has an error occurred during the initialization.
```
